projects/vett/index.js

```diff
@@ -0,0 +1,22 @@
+const sdk = require("@defillama/sdk");
+// const ethers = require("ethers");
+
+async function tvl(_timestamp, _b, { thundercore: block }) {
+  const { output: ttBalances } = await sdk.api.eth.getBalances({
+    targets: ["0xC3C857a9E5Be042C8acF4F2827Aa053e93b5d039"],
```

Comment:
one thing that is not very clear to me is, is this veTT transferable/tradable like a  erc20?

Reply:
yes, veTT is a erc20 token